html/js/tree/ISet.js

```diff
@@ -0,0 +1,151 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new ISet.
+    * @class
+    * @param {ISet} [parent] Parent ISet. If null, this is root.
+    * @param  {Number} [numOfNodes] Number of nodes of this new information set. If null 1
+    */
+    function ISet(parent, numOfNodes) {
+        this.parent = parent;
+        if (this.parent !== null) this.parent.addChild(this);
+        this.numOfNodes = numOfNodes || 1;
+         // TODO #11
+        var newNodeLevel;
+        if (parent === null) {
+            newNodeLevel = 0;
+        } else {
+            newNodeLevel = parent.node.level + 1;
+        }
+        this.node = new GTE.TREE.Node(this, newNodeLevel);
+        this.moves = [];
+    }
+
+    // TODO #12
+    /**
+    * ToString function
+    */
+    ISet.prototype.toString = function () {
+        return "ISet: " + "parent " + this.parent + "; numOfNodes: " + this.numOfNodes + "; moves: " + this.moves;
+    };
+
+    /**
+    * Function that adds a node to ISet
+    * @param {Node} node Node to add
+    */
+    ISet.prototype.addNode = function (node) {
+        this.nodes.push(node);
+    };
```

Comment:
Are we using this function? If not, I would remove it until we need it.

In any case, this.nodes is null, it hasn't been initilized (we are using by now _this.node_ only)